app/src/org/commcare/views/widgets/QuestionWidget.java

```diff
@@ -624,7 +625,10 @@ public boolean hasListener() {
     }
 
     public void checkFileSize(File file){
-        if (FileUtil.isFileOversized(file)) {
+        if (FileUtil.isFileToBigToUpload(file)) {
+            long overByAmount = (file.length() - FormUploadUtil.MAX_BYTES) / 1024;
```

Comment:
Do we think that `overByAmount` will actually be a useful/actionable piece of information for users? As opposed to just saying it's too big to upload without specifying by how much?